src/tep_sup.erl

```diff
@@ -6,14 +6,21 @@
 
 -behaviour(supervisor).
 
--export([start_link/0]).
+-export([start_link/1]).
 
 -export([init/1]).
 
 -define(SERVER, ?MODULE).
 
-start_link() ->
-    supervisor:start_link({local, ?SERVER}, ?MODULE, []).
+-spec start_link(string() | no_path) ->
+    {ok, pid()} |
+    ignore |
+    {error,
+        {already_started, pid()} |
+        {shutdown, term()} |
+        term()}.
+start_link(ConfigPath) ->
+    supervisor:start_link({local, ?SERVER}, ?MODULE, [ConfigPath]).
```

Comment:
It's actually possible to error with "already started". Seems like a good idea for our root supervisor, no?

Reply:
This is implicitly used by the calling supervisor, IRC. No need to implement it by hand, I presume.